src/users/controllers/users.controller.js

```diff
@@ -1,30 +1,76 @@
 import { usersList } from '../services/users.service.js';
+import { infoMiddleware } from '../../log-middlewares/info.log.middleware.js';
+import { errorMiddleware } from '../../log-middlewares/error.log.middleware.js';
+
+import { userMethods } from '../constants/users.constants.js';
 
 export const controllers = {
     getUsers: async (req, res) => {
-        res.json(await usersList.getUsers());
+        const { params } = req;
+        infoMiddleware(req);
```

Comment:
In general, it is not a middleware. It's just a method. A middleware is used like app.use(yourMiddleware) and has a special signature. So, I think, it's good to rename